EIPS/eip-3675.md

```diff
@@ -128,7 +128,12 @@ Changes to the block tree store that are related to the above actions **MUST** b
 
 #### EIP-2124 fork identifier
 
-For the purposes of the [EIP-2124 fork identifier](https://eips.ethereum.org/EIPS/eip-2124), nodes implementing this EIP **SHOULD** keep values of `FORK_NEXT` and `FORK_HASH` unaffected, i.e. `FORK_NEXT` is set to `0` and `FORK_HASH` retains the same value.
+For the purposes of the [EIP-2124 fork identifier](./eip-2124.md), nodes implementing this EIP **SHOULD** keep values of `FORK_NEXT` and `FORK_HASH` unaffected, i.e. `FORK_NEXT` is set to `0` and `FORK_HASH` retains the same value.
+
+Once the transition has happened, `TRANSITION_POS_BLOCK` will have some block number `N`.
+This block number cannot be known ahead of time given the dynamic nature of the transition trigger condition.
+Once this block number is known, nodes can update `FORK_HASH` by appending `uint64(N)` to the sequence of hashes used to calculate `FORK_HASH` as per EIP 2124.
```

Comment:
Isn't this the correct wording?
```suggestion
Once this block number is known, nodes MUST update `FORK_HASH` by appending `uint64(N)` to the sequence of hashes used to calculate `FORK_HASH` as per EIP 2124.
```

Reply:
Also instead of "Once this block number is known" shouldn't it just state the actual condition, i.e. "starting the block `TRANSITION_POS_BLOCK`..." ?